sdk/src/types/block/rand/mana.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::types::block::{mana::ManaAllotment, rand::output::rand_account_id};
+
+/// Generates a random mana allotment.
+pub fn rand_mana_allotment(mana: u64) -> ManaAllotment {
+    ManaAllotment::new(rand_account_id(), mana).unwrap()
```

Comment:
Why don't make the amount also random?

Reply:
I'm assuming that in future tests we'd need to provide an actual value to test edge cases

Reply:
But then just not use the random function? account id should probably also be tested in some cases

Reply:
Yeah ok, I'll change it later

Reply:
Changed